2fa-api/handlers/login.js

```diff
@@ -0,0 +1,45 @@
+const db = require("../db");
+const twilio = require("../twilio");
+
+const twilioClient = twilio.twilioClient;
+
+const login = function (req, res) {
+  try {
+    const { email } = req.body;
+    const { password } = req.body;
+
+    db.connection.query(
+      `SELECT id, phone from Users WHERE email='${email}' and password='${password}'`,
+      (error, results, _) => {
+        if (error) {
+          console.log(error);
+          return res.status(500).json({ error });
+        }
+
+        if (!results.length) {
+          return res
+            .status(200)
+            .json({ error: "incorrect email/password combo" });
+        }
+
+        // send success message
+        res.status(200).json(results[0]);
```

Comment:
what are you returning here, and why?

Reply:
Overfetching, I should send a simpler response like: `...json(loginSuccessful: true)`

Reply:
why do you even need a payload?

Reply:
do you mean just using sendStatus to communicate my intention?